engine/src/main/java/org/terasology/rendering/dag/DownSampleSceneAndUpdateExposureNode.java

```diff
@@ -0,0 +1,202 @@
+import org.terasology.rendering.assets.material.Material;
+import org.terasology.rendering.backdrop.BackdropProvider;
+import org.terasology.rendering.nui.properties.Range;
+import org.terasology.rendering.opengl.FBO;
+import org.terasology.rendering.opengl.FrameBuffersManager;
+import org.terasology.rendering.opengl.PostProcessor;
+import org.terasology.rendering.world.WorldRenderer;
+import java.nio.ByteBuffer;
+import static org.lwjgl.opengl.GL11.GL_COLOR_BUFFER_BIT;
+import static org.lwjgl.opengl.GL11.GL_DEPTH_BUFFER_BIT;
+import static org.lwjgl.opengl.GL11.glClear;
+import static org.terasology.rendering.opengl.OpenGLUtils.bindDisplay;
+import static org.terasology.rendering.opengl.OpenGLUtils.setViewportToSizeOf;
+import static org.terasology.rendering.opengl.OpenGLUtils.renderFullscreenQuad;
+
+/**
+ * TODO: Add node of this diagram
+ */
+public class DownSampleSceneAndUpdateExposureNode implements Node {
+    private static final Logger logger = LoggerFactory.getLogger(DownSampleSceneAndUpdateExposureNode.class);
+
+    @Range(min = 0.0f, max = 10.0f)
+    private float hdrExposureDefault = 2.5f;
+    @Range(min = 0.0f, max = 10.0f)
+    private float hdrMaxExposure = 8.0f;
+    @Range(min = 0.0f, max = 10.0f)
+    private float hdrMaxExposureNight = 8.0f;
+    @Range(min = 0.0f, max = 10.0f)
+    private float hdrMinExposure = 1.0f;
+    @Range(min = 0.0f, max = 4.0f)
+    private float hdrTargetLuminance = 1.0f;
+    @Range(min = 0.0f, max = 0.5f)
+    private float hdrExposureAdjustmentSpeed = 0.05f;
+
+    private float currentSceneLuminance = 1.0f;
+
+    @In
+    private WorldRenderer worldRenderer;
+
+    @In
+    private FrameBuffersManager frameBuffersManager;
+
+    @In
+    private Config config;
+
+    @In
+    private BackdropProvider backdropProvider;
+
+    @In
+    private PostProcessor postProcessor;
+
+    private RenderingConfig renderingConfig;
+    private FBO sceneOpaque;
+    private FBO scenePrePost;
+    private FBO[] downSampledScene = new FBO[5];
+    private Material downSampler;
+
+    @Override
+    public void initialise() {
+        renderingConfig = config.getRendering();
+        downSampler = worldRenderer.getMaterial("engine:prog.down");         // TODO: rename shader to downSampler
+        obtainStaticFBOs();
+    }
+
+    /**
+     * First downsamples the rendering obtained so far, after the initial post processing, into a 1x1 pixel buffer.
+     * Then calculate its pixel's luma to update the exposure value. This is used later, during tone mapping.
+     */
+    // TODO: verify if this can be achieved entirely in the GPU, during tone mapping perhaps?
+    @Override
+    public void process() {
+        sceneOpaque = frameBuffersManager.getFBO("sceneOpaque");
+        scenePrePost = frameBuffersManager.getFBO("scenePrePost");
+        if (renderingConfig.isEyeAdaptation()) {
+            PerformanceMonitor.startActivity("rendering/downsamplesceneandupdateexposure");
```

Comment:
Let's use the string "rendering/updateExposure" here. Then, in the downsampleSceneInto1x1pixelsBuffer method, let's use "rendering/updateExposure/downsampleScene" instead of Render Eye Adaptation.